pallets/gear/src/benchmarking/mod.rs

```diff
@@ -1623,6 +1640,15 @@ benchmarks! {
         verify_process(res.unwrap());
     }
 
+    mem_grow {
+        let r in 0 .. API_BENCHMARK_BATCHES;
+        let mut mem = MemoryWrap::new(DefaultExecutorMemory::new(1, None).unwrap());
+    }: {
+        for _ in 0..(r * API_BENCHMARK_BATCH_SIZE) {
+            mem.grow(WasmPage::new(1).unwrap()).unwrap();
```

Comment:
need to check that memory is succesfully growed

Reply:
But `mem.grow()` will return error if memory is failed to grow